.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
  - 6
+sudo: required
```

Comment:
Is `sudo: required` necessary? We should prefer the sudoless environment if possible.

Reply:
I'll test without

Reply:
It appears `sudo` is required. To be clear, [this runs on a machine where `sudo` is *available*](https://docs-staging.travis-ci.com/user/trusty-ci-environment/#Fully-virtualized-via-sudo%3A-required). It doesn't run everything as `root`.